lib/bap/bap_project.ml

```diff
@@ -2,6 +2,10 @@ open Core_kernel.Std
 open Bap_types.Std
 open Image_internal_std
 open Bap_disasm
+open Format
```

Comment:
We now have `Bap_image_std` and `Bap_disasm_std`, that should be opened instead of concrete modules from a library. In other words, just open this two modules and remove line 3,4,7,8